CMakeLists.txt

```diff
@@ -54,56 +50,82 @@ set(CMAKE_FIND_PACKAGE_PREFER_CONFIG ON)
 file(REMOVE_RECURSE ${CMAKE_BINARY_DIR}/pkg)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake/modules)
 
+# Provide option to enable Python support in Inviwo
+option(IVW_ENABLE_PYTHON "Enable Python support in Inviwo" ON)
+if(IVW_ENABLE_PYTHON)
+    find_package(Python3 COMPONENTS Interpreter Development NumPy)
+    if(NOT Python3_FOUND OR 
+       NOT Python3_Interpreter_FOUND OR 
+       NOT Python3_Development_FOUND OR 
+       NOT Python3_NumPy_FOUND)
+        message(FATAL_ERROR "Python3 not available and/or NumPy module not installed.\n"
+            "Please install Python3 and NumPy or disable IVW_ENABLE_PYTHON.\n"
+            "If you have a python installation that is not found, consider "
+            "setting Python3_ROOT_DIR to the root directory of your "
+            "Python 3 installation.")
+    endif()
+endif()
+
+option(IVW_ENABLE_QT "Enable Qt support in Inviwo" ON)
+if(IVW_ENABLE_QT)
+    find_package(Qt6 COMPONENTS Core)
+    if(NOT Qt6Core_FOUND) 
+        message(FATAL_ERROR
+            "Qt6 not available. Please instal Qt or disable IVW_ENABLE_QT.\n"
+            "Consider adding Qt to the CMAKE_PREFIX_PATH,\n" 
+            "i.e. start cmake with the option -DCMAKE_PREFIX_PATH=\"/path/to/qt\"")
+    endif()
+endif()
+
 include(cmake/globalconfig.cmake)
 
-#--------------------------------------------------------------------
 # Application setup
-option(IVW_APP_QTBASE       "Build base for Qt apps. Used by the Qt network editor and the Tiny Qt app" ON)
 option(IVW_APP_INVIWO       "Build Inviwo Qt network editor application" ON)
 option(IVW_APP_MINIMAL_GLFW "Build Inviwo Tiny GLFW Application" OFF)
-option(IVW_APP_MINIMAL_QT   "Build Inviwo Tiny QT Application" OFF)
+option(IVW_APP_MINIMAL_QT   "Build Inviwo Tiny QT Application" ON)
```

Comment:
any particular reason to change the previous default?

Reply:
Don't think they are needed for a basic build... if you are at the point where you need then you probably know how to turn them on? But I can be convinced any way

Reply:
I would then go for disabled by default. (To reduce build time and since they hardly will be used.)

Reply:
I thought I turned it off.. sorry for the confusion...

Reply:
will fix in the next pr...